test/pkg/firewall/firewall.go

```diff
@@ -0,0 +1,212 @@
+package firewall
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"github.com/openshift-kni/commatrix/pkg/client"
+	"github.com/openshift-kni/commatrix/pkg/consts"
+	"github.com/openshift-kni/commatrix/pkg/utils"
+
+	"github.com/openshift-kni/commatrix/pkg/types"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+func ApplyRulesToNode(cs *client.ClientSet, ns string, NFTtable []byte, nodeName string, wg *sync.WaitGroup) error {
```

Comment:
please create a struct called firewall and give him the client and the namespace.

to this function only send the nftable and the nodeName to apply

Reply:
Why is this struct needed? It's not keeping any data related to firewall. All the firewall.* funcs are internal, why applying them of the struct?
It makes the code complicated and less readable instead of just receive ns for example, the code is full of firewall.*
Another example of how the code is less readable this way
https://github.com/openshift-kni/commatrix/pull/36/files#r1726484789